pkg/mobile/mobileClient.go

```diff
@@ -66,3 +73,49 @@ func (r *MobileClientRepoImpl) DeleteByName(name string) error {
 func (r *MobileClientRepoImpl) Create(app *v1alpha1.MobileClient) error {
 	return sdk.Create(app)
 }
+
+type MobileWatcher struct {
+	events chan watch.Event
+}
+
+func (w MobileWatcher) Stop() {
+	close(w.events)
+	index := 0
+	for ; index < len(handler.watchers); index++ {
+		if handler.watchers[index] == w {
+			break
+		}
+	}
+	handler.watchers = append(handler.watchers[:index], handler.watchers[index+1:]...)
+}
+
+func (w MobileWatcher) ResultChan() <-chan watch.Event {
+	return w.events
+}
+
+type MobileHandler struct {
+	watchers []MobileWatcher
+}
+
+func (h MobileHandler) Handle(c context.Context, e sdk.Event) error {
+	for index := 0; index < len(h.watchers); index++ {
+		h.watchers[index].events <- watch.Event{
+			Type:   "MobileAppsEvent",
+			Object: nil,
+		}
+	}
```

Comment:
probably better to move the few lines of code to become the `notifyWatchers` method of the `MobileHandler`